server/apps/prepopulate/data_initialization/vocabularies.json

```diff
@@ -98,6 +98,29 @@
         ]
     },
     {
+        "_id": "priority",
```

Comment:
This is inconsistent what we have currently. Currently we have `qcode` and `name` whereas the vocabulary is using `name` and `value`.
Since we are moving the `priority` into `vocabularies` can we replace the hard-coding placed in `metadata.js`. 
Please refer https://github.com/superdesk/superdesk/blob/master/client/app/scripts/superdesk-authoring/metadata/metadata.js#L315